platform/ios/src/MGLAnnotationView.mm

```diff
@@ -37,6 +37,16 @@ - (void)setCenterOffset:(CGVector)centerOffset
     self.center = self.center;
 }
 
+- (void)setSelected:(BOOL)selected
+{
+    [self setSelected:selected animated:NO];
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated
+{
+    _selected = selected;
```

Comment:
In order to be KVO-compliant, we need to bracket this line with calls to `-willChangeValueForKey:` and `-didChangeValueForKey:`.